diff --git a/src/build/GhosttyResources.zig b/src/build/GhosttyResources.zig
index 673c7e5..ee2f657 100644
--- a/src/build/GhosttyResources.zig
+++ b/src/build/GhosttyResources.zig
@@ -25,2 +25,3 @@ pub fn init(b: *std.Build, cfg: *const Config, deps: *const SharedDeps) !Ghostty
         }),
+        .use_llvm = true,
     });
diff --git a/src/build/SharedDeps.zig b/src/build/SharedDeps.zig
index 86cfecc..bd9add5 100644
--- a/src/build/SharedDeps.zig
+++ b/src/build/SharedDeps.zig
@@ -1038,2 +1038,3 @@ pub fn gtkNgDistResources(
             }),
+            .use_llvm = true,
         });
