--- a/packages/desktop/src/main/windows/editor.ts
+++ b/packages/desktop/src/main/windows/editor.ts
@@ -105,7 +105,10 @@ class EditorWindow extends BaseWindow {
       options
     )
     if (isLinux) {
-      winOptions.icon = path.join(process.cwd(), 'static', 'logo-96px.png')
+      winOptions.icon = path.join(
+        (global as unknown as { __static: string }).__static,
+        'logo-96px.png'
+      )
     }

     const {
